<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>frmSegundoForm</class>
    <widget class="QWidget" name="frmSegundoForm">
        <property name="windowModality">
            <enum>Qt::WindowModal</enum>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>390</width>
                <height>244</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <property name="windowIcon">
            <iconset resource="../resources.qrc">
                <normaloff>:/icones/py.ico</normaloff>:/icones/py.ico
            </iconset>
        </property>
        <widget class="QPushButton" name="btnSegFormClose">
            <property name="geometry">
                <rect>
                    <x>284</x>
                    <y>210</y>
                    <width>91</width>
                    <height>24</height>
                </rect>
            </property>
            <property name="text">
                <string>Fechar</string>
            </property>
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/icones/close.ico</normaloff>:/icones/close.ico
                </iconset>
            </property>
        </widget>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>40</y>
                    <width>341</width>
                    <height>91</height>
                </rect>
            </property>
            <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;
                    color:#55aaff;&quot;&gt;Segundo Form&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;
                    font-weight:700; color:#55aaff;&quot;&gt;ApplicationModal&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                </string>
            </property>
        </widget>
    </widget>
    <resources>
        <include location="../resources.qrc"/>
    </resources>
    <connections/>
</ui>
